civihr_employee_portal/tests/HelperFunctionsTest.php

```diff
@@ -32,7 +32,8 @@ class HelperFunctionsTest extends PHPUnit_Framework_TestCase {
 
   public function testGetWhereFields() {
     $parts = [];
-    getWhereFields($this->sampleWherePart, $parts);
+    $view = new \Drupal\civihr_employee_portal\View\MyDetailsView();
+    $view->getWhereFields($this->sampleWherePart, $parts);
```

Comment:
This is confusing now, as this class is supposed to test Helper functions, but the test is checking a class method. Not only that, but a method of one specific implementation of an abstract class. What happens if I overwrite this method in `MyDetailsView`?

My suggestion here is:
- create a test class for the `AbstractView`
- move this test there
- to get an object with the method, you can mock the Abstract Class but don't mock the method itself, so the original one will be called